src/SelectSingle.vue

```diff
@@ -73,13 +76,20 @@
 				open: false,
 				searchString: '',
 				searchTimeout: null,
-				selectedIndex: undefined,
 				inputValue: ''
 			}
 		},
 		computed: {
 			activeDescendant(): string {
 				return `${this.htmlId}-item-${this.activeIndex}`
+			},
+			isDisabled(): boolean {
```

Comment:
this is kind of dumb, but I don't like that we have both `disabled` (the prop) & `isDisabled` (the computed prop that needs to consider `loading`)

Let's make it a bit more descriptive - thoughts on going full `isDisabledOrLoading`? Is that overkill?

Reply:
And importantly: should we consider making this same computed prop for `SelectMulti`, which already accepts `:loading`?

https://github.com/politico/vue-accessible-selects/blob/main/src/SelectMulti.vue#L284

Reply:
I would like to leave this opportunity (to disable based on loading) outside of component.
it's always possible to write something like `:disable="loading && someOtherCondition"`

Reply:
@jackkoppa The `isDisabledOrLoading` property name is fine for me.
@ee923925github the problem with passing `loading && someOtherCondition` in disable is impossibility to use a loading slot(to show spinner or custom message). I think more correct way will be pass this properties separately like this if you want show spinner:
```
<SingleSelect
   :disable="someOtherCondition"
   :loading="loading"
>
  <template v-slot:loadingState>
       <Spinner /> 
       Loading
  </template>
</SingleSelect>
```

Reply:
Thanks for this discussion, @ee923925github & @RED143 - I can understand the idea of handling the logic in the consuming app. But at least in the use case we have now, we do want a specific slot/placeholder text for when the component is loading, which is what requires the separate prop.

Given that `SelectMulti` already has the `loading` prop: https://github.com/politico/vue-accessible-selects/blob/main/src/SelectMulti.vue#L55-L58

I'm OK with this.

What I'd like one of us (@RED143 or @ee923925github, if either of you have capacity) to investigate in a follow-up PR is: can we make the styling behavior of `SelectSingle` & `SelectMulti` fully consistent, re: loading & disabled?

Right now after this PR, `SelectSingle` will get the same styles when disabled **or** loading. Currently, `SelectMulti` **only** gets those styles when disabled, e.g. https://github.com/politico/vue-accessible-selects/blob/main/src/SelectMulti.vue#L284

I think we should make that consistent if we can!